webpack.config.js

```diff
@@ -74,6 +75,7 @@ const webpackConfig = {
       modal: path.resolve(__dirname, 'node_modules/jquery-modal/jquery.modal'),
       raven: path.resolve(__dirname, 'node_modules/raven-js/dist/raven'),
       sinon: path.resolve(__dirname, 'node_modules/sinon/pkg/sinon'),
+      sjcl: path.resolve(__dirname, 'node_modules/sjcl/sjcl'),
```

Comment:
I don't think this is necessary; AFAICT node-jose supports all of the cryptographic operations needed.

Reply:
There was some concern expressed about the resulting bundle size if we used node-jose instead of sjcl. I think the description used was "node-jose is something of a whale." A comparison of the two libraries and the resulting bytes down the wire would be useful.